src/AcmExpandable/AcmExpandableCard/AcmExpandableCard.tsx

```diff
@@ -5,7 +5,7 @@ import { makeStyles } from '@material-ui/styles'
 
 const useStyles = makeStyles({
     toggleContainer: { alignSelf: 'center', paddingRight: '12px' },
-    cardBody: { borderTop: '1px solid rgba(0,0,0,0.1)', paddingTop: '40px' },
+    cardBody: { borderTop: '1px solid rgba(0,0,0,0.1)', paddingTop: '32px', paddingBottom: '32px' },
```

Comment:
Does this have any side-effects on other components using the `AcmExpandableCard` in this library?

Reply:
No, the only other component using `AcmExpandableCard` is `AcmCountCardSection` and that cancels the css set by AcmExpandableCard with it's makeStyles:
```
section: {
    '& > .pf-c-card__body': {
        padding: '0 !important',
    },
},
```